### DISCUSSION TITLE
Rate limit for V1 API ?

### DISCUSSION THREAD

**Discussion by yiwangethan** [acheong08/ChatGPT#883] [Q&A] [answered]
May I ask what's the rate limit per account and per address of V1 API?

**Comment by acheong08** [comment, marked as answer]
Per account is a variable by OpenAI. It is estimated to be 50-70 requests per hour. Per IP address is 50req/minute

**Reply by linhandev** [reply]
Just tried 60 req/hour and 120 req/hour on a paid account. Both didn't hit rate limit.

**Reply by acheong08** [reply]
Paid account got much higher limits

**Reply by yiwangethan** [reply]
Can I just add a parameter "paid" in the developer API to declare that this is a paid account? Like below:
chatbot = Chatbot(config={
  "email": "<your email>",
  "password": "<your password>",
  **"paid": True**
})

**Reply by linhandev** [reply]
> Can I just add a parameter "paid" in the developer API to declare that this is a paid account? Like below:
> chatbot = Chatbot(config={
>   "email": "<your email>",
>   "password": "<your password>",
>   **"paid": True**
> })

There won't be any bug if you set paid=True on unpaid account, but the model used during inference will still be the unpaid model

**Reply by yiwangethan** [reply]
Thanks for your reply. I tested a paid account, and the rate limit for a paid account is about 300 requests/hour.

**Comment by hrizal** [comment]
hmm per IP Address ,,, noted

**Reply by acheong08** [reply]
That is on the proxy server's side to limit abuse
